generate_outputs.py

```diff
@@ -143,33 +111,33 @@
 
         drive_client_wrapper.init_client_from_info(credentials_info)
 
-    # Load phone number <-> UUID table
     log.info("Loading Phone Number <-> UUID Table...")
     with open(phone_number_uuid_table_path, "r") as f:
         phone_number_uuid_table = PhoneNumberUuidTable.load(f)
 
-    # Load messages
+    log.info("Loading messages datasets:")
     messages_datasets = []
-    for i, path in enumerate(message_paths):
-        log.info("Loading Episode {}/{}...".format(i + 1, len(message_paths)))
-        with open(path, "r") as f:
-            messages_datasets.append(TracedDataJsonIO.import_json_to_traced_data_iterable(f))
-        log.debug(f"Loaded {len(messages_datasets[-1])} messages")
-
-    # Load surveys
-    log.info("Loading Demographics 1/2...")
-    with open(s01_demog_input_path, "r") as f:
-        s01_demographics = TracedDataJsonIO.import_json_to_traced_data_iterable(f)
-        log.debug(f"Loaded {len(s01_demographics)} contacts")
-
-    log.info("Loading Demographics 2/2...")
-    with open(s02_demog_input_path, "r") as f:
-        s02_demographics = TracedDataJsonIO.import_json_to_traced_data_iterable(f)
-        log.debug(f"Loaded {len(s02_demographics)} contacts")
+    for i, activation_flow_name in enumerate(pipeline_configuration.activation_flow_names):
+        raw_activation_path = f"{raw_data_dir}/{activation_flow_name}.json"
+        log.info(f"Loading {raw_activation_path}...")
```

Comment:
Consider tab indenting this log message

Reply:
after an offline discussion, not going to do this